app-services/secrets/SecretsExample.postman_collection.json

```diff
@@ -168,9 +127,11 @@
 						"v1",
 						"secret",
 						"edgex",
-						"appservice"
+						"secrets-example",
+						"mqtt"
 					]
-				}
+				},
```

Comment:
Is this still needed? It is described in the README.

Reply:
not really, most used it in debug. will remove